<launch>
	<arg name="vehicle_width" default="0.1" />
	<arg name="vehicle_length" default="0.1" />
	<arg name="horizon" default="100" />
	<arg name="min_object_size" default="0.1" />
	<arg name="max_object_size" default="30.0" />
	<arg name="polygon_quarters" default="8" />
	<arg name="polygon_resolution" default="1.0" />
	<arg name="tracking_type" default="0" /> <!-- 0 for association only, 1 for simple kf tracking, 2 for smart contour tracker -->
	<arg name="max_association_distance" default="6.5" />
	<arg name="max_association_size_diff" default="2.0" />
	
	<arg name="max_remeber_time" default="3" />
	<arg name="trust_counter" default="4" />
	<arg name="contours_circle_resolutions" default="4.0" />
	
	
	<node pkg="lidar_tracker" type="kf_contour_tracker" name="kf_contour_tracker">
       
    	<param name="vehicle_width" value="$(arg vehicle_width)" />
		<param name="vehicle_length" value="$(arg vehicle_length)" />
		<param name="horizon" value="$(arg horizon)" />
		<param name="min_object_size" value="$(arg min_object_size)" />
		<param name="max_object_size" value="$(arg max_object_size)" />
		<param name="polygon_quarters" value="$(arg polygon_quarters)" />
		<param name="polygon_resolution" value="$(arg polygon_resolution)" />
		<param name="tracking_type" value="$(arg tracking_type)" /> <!-- 0 for association only, 1 for simple kf tracking, 2 for smart contour tracker -->
		<param name="max_association_distance" value="$(arg max_association_distance)" />
		<param name="max_association_size_diff" value="$(arg max_association_size_diff)" />
		
		<param name="max_remeber_time" value="$(arg max_remeber_time)" />
		<param name="trust_counter" value="$(arg trust_counter)" />
		<param name="contours_circle_resolutions" value="$(arg contours_circle_resolutions)" />
    
  </node>

</launch>

